package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@salesforce/lwc-dev-mobile-core",
   "description": "Core module supporting Salesforce CLI mobile extension plug-ins",
-  "version": "3.2.0",
+  "version": "3.2.1",
```

Comment:
Typically, SemVer dictates that when you're adding new public APIs, that necessitates a minor version bump.

Reply:
I have another PR coming in... I'll set the version to `3.3.0` in that PR